packages/schema/src/Schema.ts

```diff
@@ -5846,6 +5847,248 @@ export class DateFromNumber extends transform(
   static override annotations: (annotations: Annotations.Schema<Date>) => typeof DateFromNumber = super.annotations
 }
 
+/**
+ * Describes a schema that represents a `DateTime.Utc` instance.
+ *
+ * @category DateTime.Utc constructors
+ * @since 0.68.26
+ */
+export class DateTimeUtcFromSelf extends declare(
+  (u) => dateTime.isDateTime(u) && dateTime.isUtc(u),
+  {
+    identifier: "DateTimeUtcFromSelf",
+    description: "a DateTime.Utc instance",
+    pretty: (): pretty_.Pretty<dateTime.Utc> => (dateTime) => dateTime.toString(),
+    arbitrary: (): LazyArbitrary<dateTime.Utc> => (fc) => fc.date().map((date) => dateTime.unsafeFromDate(date)),
+    equivalence: () => dateTime.Equivalence
+  }
+) {
+  static override annotations: (annotations: Annotations.Schema<dateTime.Utc>) => typeof DateTimeUtcFromSelf = super
```

Comment:
I removed all `static override annotations` in `main` because they interfere with tree shaking

Reply:
Did you use an alternative?

Reply:
No because they were also incorrect (it's not true that it returns a `typeof DateTimeUtcFromSelf`)